sbnanalysis/ana/SBNOsc/CMakeLists.txt

```diff
@@ -16,9 +16,21 @@ add_library(sbnanalysis_SBNOsc_NumuSelection SHARED NumuSelection.cxx)
 add_library(sbnanalysis_SBNOsc_Covariance SHARED Covariance.cxx)
 add_library(sbnanalysis_SBNOsc_Chi2Sensitivity SHARED Chi2Sensitivity.cxx)
 
+# Add in ROOT dictionary
+ROOT_GENERATE_DICTIONARY(SBNOsc_dict NumuSelection.h LINKDEF linkdef.h)
```

Comment:
See comments below about `struct NumuInteraction`.

Reply:
Still currently included for length information (and presumably going forward more stuff).